File: TP2 java/java/TV.java
public class TV extends Detail {        // inheritance dengan kelas detail
    // tambahan atribut khusus kelas tv
    private String ukuran;
    private String resolusi;
    private String tipeLayar;

    // constructor nilai awal
    public TV() {
        super();            // panggil constructor Detail
        this.ukuran = "";
        this.resolusi = "";
        this.tipeLayar = "";
    }

    // constructor isi semua data produk
    public TV(String id, String nama, int harga, String merek, String kategori, int stok,
              String ukuran, String resolusi, String tipeLayar) {
        super(id, nama, harga, merek, kategori, stok);          // panggil constructor Detail
        this.ukuran = ukuran;
        this.resolusi = resolusi;
        this.tipeLayar = tipeLayar;
    }

    // getter
    public String getUkuran() { return ukuran; }
    public String getResolusi() { return resolusi; }
    public String getTipeLayar() { return tipeLayar; }
}

File: TP2 python/python/TV.py
from Detail import Detail

class TV(Detail):
     # konstruktor dengan atribut tambahan khusus TV
    def __init__(self, id="", nama="", harga=0, merek="", kategori="", stok=0,
                 ukuran="", resolusi="", tipe_layar=""):
        super().__init__(id, nama, harga, merek, kategori, stok)        # Memanggil konstruktor detail
        self.ukuran = ukuran            # Menyimpan ukuran TV
        self.resolusi = resolusi        # Menyimpan resolusi TV
        self.tipe_layar = tipe_layar    # Menyimpan tipe layar TV

    # getter
    def get_ukuran(self): return self.ukuran
    def get_resolusi(self): return self.resolusi
    def get_tipe_layar(self): return self.tipe_layar
